### PR TITLE
#167486077 Add a migration shell script as an entry point to run migrations

### PR DESCRIPTION
**Pull request by ssewilliam**
#### What does this PR do?
- Add a shell script as an entry point to run migrations for every image that is to be built

#### Description of Task to be completed?
- Add a shell script as an entry point to run migrations for every image that is to be built

#### How should this be manually tested?
N/A

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#167486077](https://www.pivotaltracker.com/story/show/167486077)

### CODE REVIEW

**Comment by valmayaki** [comment]
You actually did more than I what I think is needed. This is another direction, test it and be sure it works. The only concern I have is that it might override the previous template that is being used, confirm that it doesn't. 
The easiest method would have been to create a shell script which would have to replace the default command, this would have included the migration script ` yarn migrate` and ` node  ./build/index.js` e.g . `yarn migrate && node  ./build/index.js`  therefore any time the container is run migration would run first before the node server.

File: scripts/entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+printf "Run database migrations"
+printf "\n======================================\n\n"
+yarn migrations
+
+printf "\n\n======================================\n"
+printf "Start the application"
+printf "\n======================================\n\n"
+node ./build/index.js
+
+exit 0
```
Comment (valmayaki) [inline scripts/entrypoint.sh]: remove the exit 0; by default `node ./build/index.js` will continuously run without reaching the `exit 0` statement

**Comment by valmayaki** [comment]
looks good

File: scripts/entrypoint.sh (hunk 1 above)
Comment (ssewilliam) [reply scripts/entrypoint.sh]: Done

**Review by Gidraff — COMMENTED** [review]
LGTM

**Pull request merged**